curriculum/challenges/english/16-the-odin-project/top-introduction-to-flexbox/introduction-flexbox-question-c.md

```diff
@@ -8,12 +8,12 @@ dashedName: introduction-flexbox-question-c
 
 Somewhat confusingly, any element can be both a flex container and a flex item. Said another way, you can also put `display: flex` on a flex item and then use flexbox to arrange its children.
 
-<img src="https://cdn.statically.io/gh/TheOdinProject/curriculum/495704c6eb6bf33bc927534f231533a82b27b2ac/html_css/v2/foundations/flexbox/imgs/04.png" />
+<img src="https://cdn.freecodecamp.org/curriculum/odin-project/flex-box/flexbox-03.png" style="width: 95%; height: 95%;" alt="This image demonstrates a nested CSS Flexbox layout. It features a larger blue rectangle. Inside the red rectangle, there are two peach-colored rectangles, identified as 'flex items'. The layout shows that a flex container can nest within another flex container, and both can have their own flex items, illustrating the concept of nesting flex containers in a Flexbox layout." />
 
 Creating and nesting multiple flex containers and items is the primary way you will be building up complex layouts. The following image was achieved using only flexbox to arrange, size, and place the various elements. Flexbox is a very powerful tool.
 
-<img src="https://cdn.statically.io/gh/TheOdinProject/curriculum/495704c6eb6bf33bc927534f231533a82b27b2ac/html_css/v2/foundations/flexbox/imgs/05.png"/>
-
+<img src="https://cdn.freecodecamp.org/curriculum/odin-project/flex-box/flexbox-02.png" alt="The image displays a CSS Flexbox layout with nested flex containers. The outer container, outlined in blue, encompasses three orange containers, each labeled as 'flex-items'. The central orange container, which also serves as a 'flex-item', contains two red containers. An arrow points to this orange container, indicating that it is not only a 'flex-item' but 'ALSO a flex container'. The red containers inside are further labeled as 'flex-items', illustrating the concept of nesting within Flexbox, where a flex item can also be a flex container."/>
```

Comment:
Is this the final image? Because the alt text and content seem to be referencing things in the image that I'm not seeing. Regardless, I don't think the alt text for this image needs to be too detailed, especially if the actual content is going to be. The intent of this image is to show that flexbox can be used to create quite complicated layouts by multiple nesting of items with containers. So I think all we need here is just a high level overview of the image that explains this, especially if the content is going to make more specific references to it. I will hold off on giving an example until I'm certain that the content and image are correct.